Tools/pixi-packages/build.sh

```
@@ -0,0 +1,37 @@
+    BUILD_DIR="../build_asan"
+    CONFIGURE_EXTRA="--with-address-sanitizer"
+    export PYTHON_ASAN="1"
+    export ASAN_OPTIONS="strict_init_order=true"
+else
+    echo "BUILD TYPE: DEFAULT"
+    BUILD_DIR="../build"
+    CONFIGURE_EXTRA=""
+fi
+
+mkdir -p "${BUILD_DIR}"
+cd "${BUILD_DIR}"
```

Shellcheck (0.11.0)

[warning] Use 'cd ... || exit' or 'cd ... || return' in case cd fails.

(SC2164)